node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Delete this file and the folder it's in, you likely ran yarn outside of the frontend directory